counsel.el

```diff
@@ -1812,11 +1812,25 @@ currently checked out."
     (define-key map [remap undo] 'counsel-find-file-undo)
     map))
 
+;;* File
+;;** `counsel-file-jump'
```

Comment:
These headings already exist in this file, and should be reused/fixed as required.

Reply:
I'd love to fix it, but I'm not sure what the file/comment conventions are. I may just as well mess it up again as correct it. Could you explain them? Or is this something you can do after the fact?

Reply:
I would normally push these changes onto your branch before merging, but as a Collaborator I don't think I have the permissions to do so.

The comments in `counsel.el` are structured like an Org-Mode tree of outlines:
```
* Top-level heading
** Sub-heading
*** Sub-sub-heading
** Another sub-heading
* Another top-level heading
```

`counsel.el` has the following top-level groups:
```
* Utility
* Async Utility
* Completion at point
* Elisp symbols
* Git
* File
* Grep
* Org
* Misc. OS
* `counsel-mode'
```

Under `* File`, you will also find `** counsel-find-file` and `** counsel-file-jump`. My suggestion is that you place your new code under the appropriate heading, and move existing headings to the right place if necessary.